near-mock-vm/assembly/context.ts

```diff
@@ -32,17 +32,17 @@ declare function _setRandom_seed(s: string): void;
 /** @internal */
 //@ts-ignore
 @external("vm", "setAttached_deposit")
-declare function _setAttached_deposit(lo: u64, hi: u64): void;
+declare function _setAttached_deposit(_u128: usize): void;
```

Comment:
@willemneal not sure how does this change work. Does it break backwards compatibility? Or old signature was misleading?

Reply:
It was an internal API. It was my attempt to pass a u128 with the high and low parts a d I went with a string instead. And this usize is the ptr to a utf8 string.